server_api.py

```diff
@@ -9,15 +9,160 @@
 import bcrypt
 import json
 import tornado
+import time
 
 from oauth2client import client
 from oauth2client import tools
 from oauth2client.file import Storage
 import httplib2
 
 from apiclient import discovery
+#from arrange_schedule import find_acticity
+#from arrange_schedule import edit_schedule
 import datetime
 #
+def check_bluetooth_mode_available():
+    file_dir = "static/user_data"
+    file_name = "setting.txt"
+    
+    #check setting file exist
+    if not os.path.exists(file_dir):
+        return 0
+    if not os.path.isfile(file_dir+'/'+file_name):
+        return 0
+    
+    try:
+        #read setting file
+        filename = file_dir + '/' + file_name
+        file_pointer = open(filename,"r")
+        bluetooth_available = 0
+        for line in file_pointer:
+            pure_data = []
+            pure_data = line.rstrip('\n').split(' ')
+            if pure_data[0] == "bluetooth_available":
+                bluetooth_available = int(pure_data[1])
+        file_pointer.close()
+        
+        #check function available
+        if bluetooth_available==1:
+            return 1
+        else :
+            return 0
+        
+        return 0
+    except:
+        return 0
+        
+#
+def find_user_by_bluetooth(db, bluetooth_id):
+    try:
+        sql = "SELECT user_id FROM user WHERE user_bluetooth_id='" + str(bluetooth_id) + "'"
+        pure_result = db.query(sql)
+        return int(pure_result[0][0])
+    except:
+        return 0
+#
+def load_now_user_prefer(user_id):
```

Comment:
According to our last discussion, the user preference data should be stored in database in stead of  stored in files.